### ISSUE TITLE
Friction: Installing Knative(docs/install/_index.md)

### ISSUE THREAD

**Issue by coryrc** [knative/docs#1969]
tl;dr On https://knative.dev/docs/install/ I believe the "Comprehensive install guides" should be the very first thing on the page, possibly relabeled "Quickstart Guides".

First line after the title "Installing Knative" is "Follow this guide to install Knative components on a platform of your choice." This is just a wordier version of the title wasting space.

It's misleading for the first paragraph to say "We provide information for installing Knative on Google Kubernetes Engine, IBM Cloud Kubernetes,.." but those don't actually link to the information on how to install Knative. I click away from the page and get confused. This whole section is redundant because the list of guides themselves say where they can be installed to. I understand you might want them in here somewhere, but it's just annoying to have it be the first thing.

Before actually getting to the install guides, there's a bunch of information about (alternative?) routing layers before getting to Istio, which has the only set of guides that actually walk you all the way through installing Knative. Given the other routing choices do not have comprehensive guides and this page doesn't explain why I should choose them, they should not be above the Istio Comprehensive guides.

It's halfway down the page, in the smallest text, before I get to "For new users, we recommend the comprehensive install to get you up and running quickly." which happens /after/ asking me to make the choice.

**Comment by samodell** [comment]
Thanks for writing this up, Cory! Totally agree with you on these points; we've got some changes coming that will hopefully address these concerns about the install sections.

**Comment by coryrc** [comment]
Woohoo! I'll look for the changes. Thanks.

**Comment by abrennan89** [comment]
@coryrc can this issue been closed now? I think the docs are pretty different now :slightly_smiling_face: https://knative.dev/docs/install/
